src/main/java/com/tagtraum/perf/gcviewer/imp/DataReaderUnifiedJvmLogging.java

```diff
@@ -401,6 +513,17 @@ private boolean lineContainsParseableEvent(ParseContext context) {
         return false;
     }
 
+    private boolean isParseablePhaseEvent(String line) {
```

Comment:
So what was happening is that [gc,phases] events were not included at all previously, when I added [gc,phases] to INCLUDE_STRINGS, it broke many existing test cases because there are [gc,phases] lines in the test files that were ignored and the test cases asserts 0 warnings. When those lines got included, the assertions failed. So this is a solution to only include ZGC [gc,phases] to not break the build and/or affect existing functionalities.